react-router-starter-template/vite.config.ts

```diff
@@ -0,0 +1,34 @@
+import autoprefixer from "autoprefixer";
+import tailwindcss from "tailwindcss";
+import tsconfigPaths from "vite-tsconfig-paths";
+import { reactRouter } from "@react-router/dev/vite";
+import { defineConfig } from "vite";
+import { cloudflare } from "@cloudflare/vite-plugin";
+
+// By default react-router's dev server uses Node.js, so we want to remove their server
```

Comment:
I'm not sure what our position on the new vite plugin is, it does mean that we end up diverging from the "stock" framework templates more than is absolutely necessary but it does mean we get to use workerd for the dev server if we make tiny adjustments to the template. Happy to remove this if this isn't the path we want to go down.

Reply:
do what is best for the person getting started